README.md

```diff
@@ -120,7 +120,8 @@ CometBFT up-to-date. Upgrading instructions can be found in
 
 Currently supported versions include:
 
-- v0.34.x: CometBFT v0.34.25 is compatible with Tendermint Core v0.34.24
+- v0.34.x: The CometBFT v0.34 series is compatible with the Tendermint Core
+  v0.34 series
```

Comment:
I know I'm getting late here but consider this:

"v0.34.x: The CometBFT v0.34 series is compatible with the Tendermint Core v0.34 series. The compatibility has been extensively tested with v0.34.26."